.github/workflows/unit-test-coverage.yml

```diff
@@ -21,9 +21,6 @@ jobs:
     permissions: write-all
     steps:
     - uses: actions/checkout@v3
-      with:
-        fetch-depth: 0
-        ref: "refs/pull/${{ github.event.number }}/merge"
```

Comment:
Copying what the other jobs are doing, idt we need to specify this. Causes more issues if we want to run on the master branch (push)